bcdevexchange/wwwroot/css/style.css

```diff
@@ -963,4 +996,51 @@ p.card-subtitle {
   .mob-space-top {
     margin-top: 4em;
   }
+}
+/*-- MW: I created these patterns for 'teasers' / These are used on homepage / We should create two view components: 'teaser-img-left' and 'teaser-img-right'; --*/

```

Comment:
again, we shouldn't need the name of who committed something (we can use git blame to find that out)
/*--------Begin Home page teaser styling------------*/

/*------------End Home page teaser styling----------------*/

Reply:
Right 🙂  I will clean up.